src/arrayMethodLastIndexOf.js

```diff
@@ -5,8 +5,23 @@
  *
  */
 function applyCustomLastIndexOf() {
-  [].__proto__.lastIndexOf2 = function() {
-    // write code here
+  [].__proto__.lastIndexOf2 = function(token, index = 0) {
+    let lastIndex;
+    if (index < 0) {
+      index = index + this.length;
```

Comment:
Do not mutate function arguments create another variable(ideally create constants and do not reassign them)